frontend/src/__tests__/cypress/cypress/utils/pagination.ts

```diff
@@ -0,0 +1,52 @@
+import { tablePagination } from '~/__tests__/cypress/cypress/pages/components/Pagination';
+
+type PaginationProp = {
+  totalItems: number;
+  firstElement: string;
+  paginationVariant: string;
+};
+
+export const testPagination = ({
+  totalItems = 0,
+  firstElement = 'Test',
+  paginationVariant = 'top',
```

Comment:
is the default value needed here?

Reply:
I think, those are not required. I forgot to remove those. I will remove that.